maquetado/scss/tech.scss

```diff
@@ -0,0 +1,68 @@
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+%card {
+	height: 20em;
+	border: 1px solid $green;
+	border-radius: 5px;
+	margin: 15px 15px;
+	width: 14em;
+}
+
+.container {
+	display: flex;
+	justify-content: center;
+	width: 80%;
+}
+
+.cards {
+	display: flex;
+	flex-wrap: wrap;
+	width: 80%;
```

Comment:
same, use fixed value, different screens may change the elements shape and you dont want that